src/witan/app/schema.clj

```diff
@@ -44,9 +44,9 @@
 (def ModelPropertyType
   "Types that the model properties can be"
   (s/enum
-   "text"
-   "number"
-   "dropdown"))
+   :text
+   :number
+   :dropdown))
```

Comment:
A recall changing these from keys to strings - was it intentional to change them back?